definitions/rest-definitions.d.ts

```diff
@@ -13,10 +13,12 @@ declare module "rest-definitions" {
 
     export interface DeploymentStatusReport {
         appVersion: string;
-        clientUniqueId: string;
-        deploymentKey: string; 
+        clientUniqueId?: string;
+        deploymentKey: string;
+        previousDeploymentKey?: string;
+        previousLabelOrAppVersion?: string;
```

Comment:
may be create an object to aggregate these properties? If you think it makes sense.
```
export interface PreviousAppConfiguration
   {
        previoustDeploymentKey?: string;
        previousLabelOrAppVersion?: string;
   }
```